eng/Version.Details.xml

```diff
@@ -108,6 +108,11 @@
       <Uri>https://github.com/dotnet/fsharp</Uri>
       <Sha>f8d54670c81ec6e29d312867207a38d0dc3aa192</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.fsharp" Version="5.0.2-beta.21321.2" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
The Microsoft.FSharp.Compiler cannot be adorned as source-build as it's version does not match the SB intemerdiate package version therefore the SB Intermediate dependency must be explicitly added.